word2vec_decade_experiment/02_word2vec_decade_analysis.py

```diff
@@ -0,0 +1,73 @@
+# ---
+# jupyter:
+#   jupytext:
+#     formats: ipynb,py
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.9.1+dev
+#   kernelspec:
+#     display_name: Python [conda env:biovectors]
+#     language: python
+#     name: conda-env-biovectors-py
+# ---
+
+# # Analyze Word2Vec by Decades Run
+
+# +
+from pathlib import Path
```

Comment:
Here you're looking to compare how well similarity scores of (gene, disease) pairs correlates/predicts known (gene, disease) pairs annotated in hetionet?

Reply:
>Here you're looking to compare how well similarity scores of (gene, disease) pairs correlates/predicts known (gene, disease) pairs annotated in hetionet?

Here we are looking at how gene-disease pairs correlate based on abstracts of a given time period. We wanted to see what pairs were popular given the decade. The hetionet labeling is to see if the pair was already discovered or not as undiscovered pairs would generate new hypotheses for geneticists.